mypy/fscache.py

```diff
@@ -97,12 +97,39 @@ def listdir(self, path: str) -> List[str]:
         return results
 
     def isfile(self, path: str) -> bool:
-        st = self.stat(path)
-        return stat.S_ISREG(st.st_mode)
+        try:
+            st = self.stat(path)
+            return stat.S_ISREG(st.st_mode)
+        except OSError:
+            return False
+
+    def isfile_case(self, path: str) -> bool:
+        """Return whether path exists and is a file.
+
+        On case-insensitive filesystems (like Mac or Windows) this returns
+        False if the case of the path's last component does not exactly
+        match the case found in the filesystem.
```

Comment:
Hm, this means that importing wrongly-cased *packages* will not be flagged, since we only compare the filename (which would be `__init__.py` for a package). Not actionable except perhaps add a TODO comment; this was a pre-existing problem. (Python itself gets it right so that's a fallback.)